survey-runner.tf

```diff
@@ -61,10 +62,11 @@ module "eq-ecs" {
 }
 
 module "survey-runner-on-ecs" {
-  source                         = "github.com/ONSdigital/eq-survey-runner-deploy"
+  source                         = "github.com/ONSdigital/eq-survey-runner-deploy?ref=define-aws-version"
```

Comment:
If I merge the other PRs first, this shouldn't rely on a branch?

Reply:
Correct. i left these in for easy testing